templates/myaccount/view-order-information.php

```diff
@@ -0,0 +1,167 @@
+					<td>
+						<?php echo $order->get_price( 'trial_total' ); ?>
+						<?php printf( _n( 'for %1$d %2$s', 'for %1$d %2$ss', $order->get( 'trial_length' ), 'lifterlms' ), $order->get( 'trial_length' ), $order->get( 'trial_period' ) ); ?>
+					</td>
+				</tr>
+			<?php endif; ?>
+
+			<?php if ( $order->has_discount() ) : ?>
+				<tr>
+					<th><?php _e( 'Original Total', 'lifterlms' ); ?></th>
+					<td><?php echo $order->get_price( 'original_total' ); ?></td>
+				</tr>
+
+				<?php if ( $order->has_sale() ) : ?>
+					<tr>
+						<th><?php _e( 'Sale Discount', 'lifterlms' ); ?></th>
+						<td>
+							<?php echo $order->get_price( 'sale_price' ); ?>
+							(<?php echo llms_price( $order->get_price( 'sale_value', array(), 'float' ) * -1 ); ?>)
+						</td>
+					</tr>
+				<?php endif; ?>
+
+				<?php if ( $order->has_coupon() ) : ?>
+					<tr>
+						<th><?php _e( 'Coupon Discount', 'lifterlms' ); ?></th>
+						<td>
+							<?php echo $order->get_coupon_amount( 'regular' ); ?>
+							(<?php echo llms_price( $order->get_price( 'coupon_value', array(), 'float' ) * - 1 ); ?>)
+							[<a href="<?php echo get_edit_post_link( $order->get( 'coupon_id' ) ); ?>"><?php echo $order->get( 'coupon_code' ); ?></a>]
+						</td>
+					</tr>
+				<?php endif; ?>
+			<?php endif; ?>
+
+			<tr>
+				<th><?php _e( 'Total', 'lifterlms' ); ?></th>
+				<td>
+					<?php echo $order->get_price( 'total' ); ?>
+					<?php if ( $order->is_recurring() ) : ?>
+						<?php
+						printf(
+							// Translators: %1$d = the billing frequency; %2$s = the billing period.
+							_n( // phpcs:ignore: WordPress.WP.I18n.MismatchedPlaceholders -- It's not an error.
+								'Every %2$s', // phpcs:ignore: WordPress.WP.I18n.MissingSingularPlaceholder -- It works as expected despite the CS warning.
+								'Every %1$d %2$ss',
+								$order->get( 'billing_frequency' ),
+								'lifterlms'
+							),
+							$order->get( 'billing_frequency' ),
+							$order->get( 'billing_period' )
+						);
+						?>
+						<?php if ( $order->get( 'billing_cycle' ) > 0 ) : ?>
+							<?php printf( _n( 'for %1$d %2$s', 'for %1$d %2$ss', $order->get( 'billing_cycle' ), 'lifterlms' ), $order->get( 'billing_cycle' ), $order->get( 'billing_period' ) ); ?>
+						<?php endif; ?>
+					<?php else : ?>
+						<?php _e( 'One-time', 'lifterlms' ); ?>
+					<?php endif; ?>
+				</td>
+			</tr>
+
+			<tr>
+				<th><?php _e( 'Payment Method', 'lifterlms' ); ?></th>
+				<td>
+					<?php if ( is_wp_error( $gateway ) ) : ?>
+						<?php echo $order->get( 'payment_gateway' ); ?>
+					<?php else : ?>
+						<?php echo $gateway->get_title(); ?>
+					<?php endif; ?>
+					<?php do_action( 'lifterlms_view_order_after_payment_method', $order ); ?>
+				</td>
+			</tr>
+
+			<tr>
+				<th><?php _e( 'Start Date', 'lifterlms' ); ?></th>
+				<td><?php echo $order->get_date( 'date', 'F j, Y' ); ?></td>
+			</tr>
+			<?php if ( $order->is_recurring() ) : ?>
```

Comment:
Perhaps we should pass in the WordPress date format to the this template instead of hard coding it?
```php
$date_format = get_option( 'date_format' );
```
Or maybe `LLMS_Post_Model::get_date_format()`.

This applies to lines 137, 142, 150, and 162.

Reply:
I'm not interested in addressing this at this moment, we're already tracking date issues in #392 

I want to address this in a better way than we currently are and at this moment we don't need to invest the time in the rabbit hole.

Reply:
Specifically this issue is: https://github.com/gocodebox/lifterlms/issues/1690